docs/spec/fungible-token-transfer/Bank.tla

```diff
@@ -0,0 +1,89 @@
+-------------------------------- MODULE Bank --------------------------------
+
+EXTENDS Integers, FiniteSets
+
+\* subtract coins from account
+SubtractCoins(accounts, accountID, amount) ==
```

Comment:
I think we will need to add denoms also to the picture. So when you subtract you also need to specify denom as accounts are multi-denomed one.